src/TDB2.h

```diff
@@ -30,25 +30,21 @@
 #include <FS.h>
 #include <Task.h>
 #include <stdio.h>
-#include <tc/Replica.h>
-#include <tc/WorkingSet.h>
+#include <taskchampion-cpp/lib.h>
 
 #include <map>
+#include <optional>
```

Comment:
This was included by one of the `tc/*.h`, masking the failure to include it directly in this file ([include-what-you-use](https://include-what-you-use.org/))